src/Main.ts

```diff
@@ -826,6 +828,27 @@ export class Main {
         }
         const port = this.config.homeserver.appservice_port || cliPort;
         this.bridge.run(port, this.config, this.appservice);
+
+        this.bridge.addAppServicePath({
+            handler: this.onHealthProbe.bind(this.bridge),
+            method: "GET",
+            path: "/health",
+            checkToken: false,
+        });
+
+        this.bridge.addAppServicePath({
+            handler: this.onReadyProbe.bind(this.bridge),
+            method: "GET",
+            path: "/ready",
+            checkToken: false,
+        });
+
+        this.stateStorage = new StateLookup({
```

Comment:
I don't understand what paul was warning against and why this does not apply anymore.

Reply:
It was about the state storage not being ready until after the bridge had started, but tbh I don't think it's been a problem for us in a long time.